src/components/Header/Devices/desktop.tsx

```diff
@@ -0,0 +1,98 @@
+import {
+  Box,
+  Flex,
+  Link,
+  Stack,
+  Text,
+  Popover,
+  PopoverContent,
+  PopoverTrigger,
+  useDisclosure
+} from "@chakra-ui/react";
+import { HeaderProps, Links } from "../";
+
+  /* Connect Wallet Options */
+export function DesktopSubNav(props: HeaderProps) {
+
+  const { name, href } = props;
+
+  return (
+    <>
+      <Link
+        href={href}
+        role={'group'}
+        display={'block'}
+        p={2}
+        rounded={'md'}
+        _hover={{ bg: 'navbar.marketplace' }}>
+        <Stack direction={'row'} align={'center'}>
+          <Box>
+            <Text
+              transition={'all .3s ease'}
+              _groupHover={{ color: 'brown' }}
+              fontWeight={500}>
+              {name}
+            </Text>
+          </Box>
+          <Flex
+            transition={'all .3s ease'}
+            transform={'translateX(-10px)'}
+            opacity={0}
+            _groupHover={{ opacity: '100%', transform: 'translateX(0)' }}
+            justify={'flex-end'}
+            align={'center'}
+            flex={1}>
+          </Flex>
+        </Stack>
+      </Link>
+    </>
+    );
+  };
+
+export default function DesktopNav() {
+
+  const { isOpen } = useDisclosure();
+
+  return (
+    <>  
+      {/* Stacking */}
+      <Stack direction={'row'} spacing={4}>
+        {/* Options */}
+        {Links.map((link) => (
+        <Box key={link.name}>
+          <Popover trigger={'hover'} placement={'bottom-start'} >
+            <PopoverTrigger>
+              <Link
```

Comment:
If you're using NextJS you should use `next/link` component to navigate through the app
If you a normal `<a>` you force your all app to refresh
try it and see the difference

https://nextjs.org/docs/api-reference/next/link

Reply:
Done!